pinot-core/src/main/java/com/linkedin/pinot/core/plan/maker/InstancePlanMakerImplV2.java

```diff
@@ -40,16 +41,17 @@
 public class InstancePlanMakerImplV2 implements PlanMaker {
   private static final Logger LOGGER = LoggerFactory.getLogger(InstancePlanMakerImplV2.class);
 
+  // TODO: Fix the runtime trimming and add back the number of aggregation groups limit.
```

Comment:
I take it that you chose to comment these lines as opposed to removing them is because we want to revisit this part? If so, please state that as part of the comments.